src/public/main.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let registrationData;
+let occupations;
+let stateOptions;
+let occupationList = document.getElementById('occupationList');
+let statesList = document.getElementById('statesList');
+
+const populateOccupations = () => {
+  for (let i = 0; i < occupations.length; i++) {
+    occupationList.options[i] = new Option(`${occupations[i]}`);
+  }
+};
+
+const populateStates = () => {
+  for (let i = 0; i < stateOptions.length; i++) {
+    statesList.options[i] = new Option(`${stateOptions[i].name}`);
+  }
+};
+
+window.addEventListener('load', () => {
+  fetch('https://frontend-take-home.fetchrewards.com/form')
+  .then(response => response.json())
+  .then(data => registrationData = data)
+  .then(data => occupations = registrationData.occupations)
+  .then(data => stateOptions = registrationData.states)
+  .then(() => populateOccupations())
+  .then(() => populateStates())
+  .then(() => console.log(stateOptions))
+});
+
```

Comment:
This event listener fetches the drop-down option data before assigning them to their respective variables. Once this has been done, **populateOccupations** and **populateStates** are run to populate the drop-down menus.

(The console log was just to view the structure of the data as it came back, and is not present in the final version of this repo.)